src/modules/core/components/TokenEditDialog/TokenItem/TokenItem.tsx

```diff
@@ -19,14 +15,10 @@ const MSG = defineMessages({
 });
 
 interface Props {
-  nativeTokenAddress?: Address;
-  removeTokenFn: (address: Address) => void;
   token: AnyToken;
 }
 
-const TokenItem = ({ nativeTokenAddress, removeTokenFn, token }: Props) => {
-  const canRemoveToken =
-    nativeTokenAddress !== token.address && !tokenIsETH(token);
```

Comment:
We'll still need this check but in a different form. I'm just mentioning this so I don't forget.

Nothing to do here :)